1039-wizard-design-guidance.md

```diff
@@ -0,0 +1,31 @@
+---
+category: component design guidance
+state: draft
+created: 2024-10-17
+updated: 2024-10-22
+---
+
+# Wizard Design Guidance
+
+The [wizard](https://clarity.design/documentation/wizard) component guides the user through a set of predefined steps. Wizards can help users focus on only the input needed for each step and decrease the chance of errors.
+
+## Guidance
+
+### Wizard Usage
+
+- **Must** have at least two steps in your wizard. 
+- **Must not** have long or complex content in each step of the wizard.
```

Comment:
```suggestion
- Design teams **must not** include long or complex content in each step of the wizard, as this can overwhelm users and hinder their progress. Each step should be clear and concise.
```